int64 timestamp
float64 longitude
float64 latitude
float64 heading